content/en/docs/releases/supported-releases/index.md

```diff
@@ -51,16 +51,17 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported  | Release Date      | End of Life            | Supported Kubernetes Versions | Tested, but not supported |
-|-----------------|----------------------|-------------------|------------------------|-------------------------------|---------------------------|
-| master          | No, development only |                   |                        |                               |                           |
-| 1.12            | Yes                  | November 18, 2021 | ~June 2022 (Expected)  | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18          |
-| 1.11            | Yes                  | August 12, 2021   | ~Mar 2022 (Expected)   | 1.18, 1.19, 1.20, 1.21, 1.22  | 1.16, 1.17                |
-| 1.10            | No                   | May 18, 2021      | Dec 30, 2021           | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17, 1.22          |
-| 1.9             | No                   | February 9, 2021  | Oct 8, 2021            | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                |
-| 1.8             | No                   | November 10, 2020 | May 12, 2021           | 1.16, 1.17, 1.18, 1.19        | 1.15                      |
-| 1.7             | No                   | August 21, 2020   | Feb 25, 2021           | 1.16, 1.17, 1.18              | 1.15                      |
-| 1.6 and earlier | No                   |                   |                        |                               |                           |
+| Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported |
+|-----------------|----------------------|-------------------|--------------------------|-------------------------------|---------------------------|
+| master          | No, development only |                   |                          |                               |                           |
+| 1.13            | Yes                  | February 8, 2022  | ~October 2022 (Expected) | 1.20, 1.21, 1.22, 1.23        | 1.16, 1.17, 1.18, 1.19        |
```

Comment:
Might have one extra space in last columns for a few lines (or it's my browser), and update the date as well.

Reply:
Thanks! I have fixed the extra spaces in last column and the date.